src/test/units/AppFormValidators.test.js

```diff
@@ -135,7 +135,7 @@ describe("App Form Validators", function () {
 
       it("list of valid constraints", function () {
         let constraints =
-          "hostname:UNIQUE, atomic:LIKE:man, rackid:CLUSTER:rack-1";
+          "hostname:UNIQUE, atomic:LIKE:man, rackid:CLUSTER:rack-1, dc:max_per:2";
```

Comment:
Could you write `max_per` in capital letters here? Just for consistency reasons in this test.
Thank you so much.